xbmc/pvr/addons/PVRClient.cpp

```diff
@@ -270,12 +270,15 @@ void CPVRClient::WriteClientTimerInfo(const CPVRTimerInfoTag &xbmcTimer, PVR_TIM
 
   addonTimer.iClientIndex      = xbmcTimer.m_iClientIndex;
   addonTimer.state             = xbmcTimer.m_state;
+  addonTimer.type              = xbmcTimer.m_type;
+  addonTimer.iClientIndex      = xbmcTimer.m_iClientIndex;
```

Comment:
This line can be removed now, the assignment a few lines above accidentally got removed but has now been added back in.